DNN Platform/Library/Entities/Profile/ProfileController.cs

```diff
@@ -602,11 +602,11 @@ private static string RemoveUnsafeAttributes(string content)
                 needEncode = true;
             }
 
-            var tagMatches = Regex.Matches(cleanContent, @"(<[^>]*?) on.*?\=(['""]?)[\s\S]*?(\2 )([^>]*?>)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+            var tagMatches = Regex.Matches(cleanContent, @"(<[^>]*?) on.*?\=(['""]?)[\s\S]*?(\2 )([^>]*?>)", RegexOptions.IgnoreCase);
```

Comment:
Compiling Regex components that are created on the fly is very slow and keeps adding to the temporary IL generated code. I did few tests and compiling these (non-static) ones is 3 to 6 times slower than non compiled ones.